src/include/ipc4/copier.h

```diff
@@ -246,7 +240,7 @@ struct copier_data {
 
 	/* buffer to mux/demux data from/to multiple endpoint buffers for ALH multi-gateway case */
 	struct comp_buffer *multi_endpoint_buffer;
-	channel_copy_func copy_single_channel;
+	pcm_converter_func copy_single_channel;
```

Comment:
simply change the function type? can we further assign pcm converter function instead of use above 2 function?
i.e, use audio_stream_copy in pcm converter to cover this?